test/bin/build_images.sh

```diff
@@ -214,6 +214,19 @@ do_group() {
         fi
     done
 
+    echo "Checking build statuses"
```

Comment:
Instead of looping again, the variable could be set up around line 193-196 where the status condition is checked already and then this could just be a test and return.